test/e2e/admission_controller/admission_controller_test.go

```diff
@@ -34,9 +34,11 @@ import (
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/common"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/k8s"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/yunikorn"
+
+	siCommon "github.com/apache/yunikorn-scheduler-interface/lib/go/common"
```

Comment:
Using `constants.DomainYuniKorn` not import SI

Reply:
Thank you, update the commit.